k8s/aat/cluster-00-overlay/wa/kustomization.yaml

```diff
@@ -6,8 +6,10 @@ bases:
   - ../../../namespaces/wa/wa-task-batch-initiation/wa-task-batch-initiation.yaml
   - ../../../namespaces/wa/wa-task-batch-termination/wa-task-batch-termination.yaml
   - ../../../namespaces/wa/wa-task-batch-ad-hoc-pending-termination/wa-task-batch-ad-hoc-pending-termination.yaml
+  - ../../../namespaces/wa/wa-message-find-problem-messages/wa-message-find-problem-messages.yaml
```

Comment:
I would pluralise the prefix we are using i.e. `wa-messages-. . .`

Reply:
done